### PR TITLE
Adding Multiple Table Rendering Feature

### PR DESCRIPTION
**Pull request by OzanKurt**
Hi, I added support for rendering 2 tables in a single page.

The idea was pretty simple. 

I had 2 tables in `/users`, `ActiveUsersDataTable` and `InactiveUsersDataTable`.

- I added and `id` field to use in html part of the datatable so that the ids won't conflict.
- I added `?tableId=TableID` to each request so that we can define which table we're querying with the ajax request to the backend.
- I edited the `server-side.buttons.js` to allow urls with a query parameter.

### CODE REVIEW

**Comment by yajra** [comment]
Note: This is in connection with https://github.com/yajra/laravel-datatables-buttons/pull/5

> I added ?tableId=TableID to each request so that we can define which table we're querying with the ajax request to the backend.

I think we should make this parameter optional. The PR will always add this parameter which is not needed by some of the users. This is also the reason why the test fails.

**Comment by OzanKurt** [comment]
Yeah that can be optional for sure. Would you like to have a look yourself, or should I try?

**Comment by yajra** [comment]
Can you please try first. Having a lot of incoming task lately and I may not be able to look at this asap. Thanks!

**Comment by OzanKurt** [comment]
@yajra Done some changes.

**Comment by yajra** [comment]
@OzanKurt thanks for the changes. Will try to check and release this within week.

File: src/Html/Builder.php [hunk 1]
```diff
@@ -538,11 +563,20 @@ public function addIndex(array $attributes = [])
...
     public function pipeline($url, $pages)
     {
+        $url .= $this->getQueryString();
+
         $this->ajax = "$.fn.dataTable.pipeline({ url: '{$url}', pages: {$pages} })";
 
         return $this;
     }
 
+    public function getQueryString()
```
Comment (yajra) [inline src/Html/Builder.php]: Sorry for late response, I think this part may cause some breaking change? What if we already have a query string on our default URL? 

```php
'ajax' => '/data?user_id=1',
...
```

**Comment by OzanKurt** [comment]
i believe added an exceptional functionality for that too, let me have a look and fix if not.

Ozan Kurt

**Comment by yajra** [comment]
Sorry for overlooking this. Please submit a new PR if still interested to have this. Thanks!

File: src/Html/Builder.php (hunk 1 above)
Comment (OzanKurt) [reply src/Html/Builder.php]: Exactly, I had that issue and also updated accordingly. I am currently so overwhelmed by projects, sadly don't have time to re-think some code and push them.

**Pull request closed without merging**